### DISCUSSION TITLE
Ability to log on so that the translate history is visible

### DISCUSSION THREAD

**Discussion by Stefandr0** [ThijmenDam/BarTranslate#37] [Ideas]
Would be great if I could log in with my G account and have my translate history available.

First time I opened the app there appeared a login menu but i skipped it. Now I can't find it  to log in....

Otherwise great app - THANK YOU! <3

**Comment by ThijmenDam** [comment]
Glad you like BarTranslate @Stefandr. Good suggestion too! I'll create a GitHub issue from this topic, and tag it as a feature request. See https://github.com/ThijmenDam/BarTranslate/issues/38. We can continue the discussion there :-)

As for the implementation, I think a "log in to Google" (e.g. in the settings pane of BarTranslate) that opens a new log-in window should do the job, don't you think?
